common/src/main/java/org/embeddedt/modernfix/common/mixin/perf/cache_profile_texture_url/MinecraftProfileTextureMixin.java

```diff
@@ -16,29 +15,27 @@
 
 @Mixin(value = MinecraftProfileTexture.class, remap = false)
 @ClientOnlyMixin
-public abstract class MinecraftProfileTextureMixin implements ICachedProfileTexture {
+public abstract class MinecraftProfileTextureMixin {
 	/**
 	 * @author Fury_Phoenix
-	 * @see org.embeddedt.modernfix.common.mixin.perf.cache_profile_texture_url.YggdrasilGsonDeserializerMixin#setCachedURL
+	 * This is deserialized normally. @see MinecraftProfileTexture#getHash
 	 **/
-	
+	@Shadow @Final private String url;
```

Comment:
This string is making Gradle complain that it's symbl could not be found.

Reply:
Humor me I probably forgot to capitalize it right...

Reply:
No, wait, forgot an import.